dabest/_classes.py

```diff
@@ -28,6 +28,7 @@ def __init__(self, data, idx, x, y, paired, id_col, ci, resamples,
         self.__idx         = idx
         self.__id_col      = id_col
         self.__is_paired   = paired
+        self.__is_preceding= preceding
```

Comment:
Please change `preceding` to `repeated_measures` and `__is_preceding` to `__ repeated_measures`